app/src/main/java/com/edulectronics/tinycircuit/Views/CircuitActivity.java

```diff
@@ -312,8 +313,10 @@ private void giveNegativeFeedback(){
     //Create a positive feedback message
     private void givePositiveFeedback(){
         String[] positiveFeedback = getResources().getStringArray(R.array.positive_feedback);
+        String feedback = giveFeedback(positiveFeedback) + " " + getResources().getString(
```

Comment:
Ik zou deze logica helemaal verplaatsen naar de levelcontroller, of zelfs het scenario. Niet hier in ieder geval.